src/main/ml-modules/services/sm-match.xqy

```diff
@@ -55,7 +55,10 @@ function post(
       $options//*:max-scan ! xs:integer(.),
       20
     ))
-  let $include-matches := fn:head((map:get($params, "includeMatches"), fn:false()))
+  let $include-matches := 
```

Comment:
As written, this would quietly ignore bad format values. Please update to something like this:

```
let $include-matches as xs:boolean := 
  let $include := fn:head(map:get($params, "includeMatches"), fn:false())
  return
    if ($include castable as xs:boolean) then
      $include
    else 
      fn:error((),"RESTAPI-SRVEXERR", 
        (400, "Bad Request", 
         "includeMatches must be a boolean: " || $include))
```